conftest.py

```diff
@@ -0,0 +1,42 @@
+"""Place Pytest fixtures to be used across all tests here."""
+import pytest
+import ray
+
+
+@pytest.fixture(scope='module')
+def ray_local_test_fixture():
+    """Initializes Ray and shuts down Ray in local mode.
+
+    Yields:
+        None: Yield is for purposes of pytest module style.
+            All statements before the yield are apart of module setup, and all
+            statements after the yield are apart of module teardown.
+    """
+    if not ray.is_initialized():
+        ray.init(local_mode=True,
+                 ignore_reinit_error=True,
+                 log_to_driver=False,
+                 include_webui=False)
+    yield
+    if ray.is_initialized():
+        ray.shutdown()
+
+
+@pytest.fixture(scope='module')
+def ray_test_fixture():
```

Comment:
i don't think "test_fixture" is really needed here. `ray` is probably the most canonical name but makes a name collision, so maybe you can use `ray_session` and `ray_local_session`?